src/v3/src/transformer/terminal/transformTerminalTransaction.ts

```diff
@@ -158,6 +160,7 @@ export const transformTerminalTransaction = (
     const shouldRedirect = isOauth2Enabled(widgetProps) === false || widgetProps.redirect === 'always';
     if (shouldRedirect) {
       SessionStorage.removeStateHandle();
+      removeUserAuthenticatedCookie();
```

Comment:
Similar question, why remove the cookie here if this is when the user is authenticated?

Reply:
cleaned up: https://github.com/okta/okta-signin-widget/pull/3434/commits/e0ff2c0d00c8b9d7301fde3c0bb75d9b516ccbd9